contingency/contingency-api/src/test/resources/contingencies.json

```diff
@@ -1,4 +1,5 @@
 {
+  "type" : "default",
```

Comment:
Note that we should try to ensure retro-compatibility, so it should work without this